crates/server/src/api/routes/quests/mod.rs

```diff
@@ -67,3 +69,29 @@ pub fn get_user_address_from_request(req: &HttpRequest) -> Result<String, HttpRe
         Err(HttpResponse::BadRequest().body("Bad Request"))
     }
 }
+#[allow(clippy::invalid_regex)]
```

Comment:
why it's invalid regex? shouldn't we use an URL parser instead of a regex?

Reply:
It's a known clippy bug, it's a valid regex. I prefer to use the regex instead of adding a new crate like `url` which adds a lot of stuff that we are not gonna use it.